### DISCUSSION TITLE
阿里云盘无法上传

### DISCUSSION THREAD

**Discussion by 17603015069** [AlistGo/alist#3447] [Q&A]
![image](https://user-images.githubusercontent.com/125480035/219252036-16c15773-f51b-4023-a9c9-cc1197acbcdb.png)
报错内容如下：Put "http://ccp-bj29-bj-1592982087.oss-cn-beijing-internal.aliyuncs.com/YcvQ643r%2F65955272%2F63ed9007b1684ee8290a481d8b179e2fe489af57%2F63ed9007530a0455197147509274d6e9be619ae9?partNumber=1&security-token=CAIS%2BgF1q6Ft5B2yfSjIr5flE9DavbwW2ZqZM1TQtlFkP%2BxtvfGStDz2IHFPeHJrBeAYt%2FoxmW1X5vwSlq5rR4QAXlDfNUmBAAfaqVHPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLukMWr%2Fwq3PIdp2ma447NWQlLnzyCMvvJ9OVDFyN0aKEnH7J%2Bq%2FzxhTPrMnpkSlacGoABBbaaqYBBesNvn41%2BGxveSOcEY8VrvQKO4sUCZx6Rvj%2Fxgqd57ozfrToa2FL1q5DGm98UqCz18%2BTO9OpDMYAqvet%2BvRJGsDuh5neUqTOPpbLVN%2FIKAvnH56phQPGkp1eQTLawwglHh09yrIwMdIeqsxemgXPA%2Bb0%2BSDNdBiEGt2M%3D&uploadId=6CC2459C714F4DF2BE73DF18682758E6&x-oss-access-key-id=STS.NTPXjnPc7nXr1raRQ13cAR49V&x-oss-expires=1676516887&x-oss-signature=VfO7H8rZ%2Bj97Og6TQ9sT67SOYEwtBvolK1hvmimBiYk%3D&x-oss-signature-version=OSS2": dial tcp *************2:80: i/o timeout

**Comment by Dashan-IZ** [comment]
试试用ip+端口能上传吗？如果能可能就是你的nginx的配置有问题，

有其他问题可以加入qq群聊，大佬多多，有问必答：746260284
